src/main/java/seedu/address/model/customer/Customer.java

```diff
@@ -52,6 +64,15 @@ public Customer(Name name, Phone phone, Email email, Address address, Set<Tag> t
         this.tags.addAll(tags);
     }
 
+    /**
+     * Initialises Customer with a list of commissions.
+     */
+    public Customer(Name name, Phone phone, Email email, Address address, Set<Tag> tags,
```

Comment:
Just some thought I had - wondering if there's a better way to handle all the different constructor variations, considering using something like [builder pattern](https://hudsonmendes.medium.com/immutable-the-builder-pattern-vs-messy-pojos-or-pocos-e6b5be894a9d)